test/parallel/test-assert.js

```diff
@@ -587,8 +587,9 @@ function testBlockTypeError(method, block) {
     method(block);
     threw = false;
   } catch (e) {
-    assert.strictEqual(e.toString(),
-                       'TypeError: "block" argument must be a function');
+    assert.strictEqual(
+      e.toString(),
+      'TypeError[ERR_ARG_FUNCTION]: "block" argument must be a function');
```

Comment:
Looking at this more closely, it seems like there is an intentional avoidance of `assert.throws()` since `assert.throws()` is one of the things being tested here. (So it avoids using `assert.throws()` to confirm that `assert.throws()` throws.)

And without using `assert.throws()`, the appeal of using `common.expectsError()` decreases.